conf/apt.conf.d/99clean

```diff
@@ -0,0 +1,7 @@
+# Keep the post installation environment clean
+# and completely empty the cache after upgrade/update
+# for more options see: man (5) apt.conf 
+
+DPkg::Post-Invoke { "echo Kano cache cleanup...;rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
+APT::Update::Post-Invoke { "echo Kano post cleanup...;rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
```

Comment:
Don't these `echo`s need the string to be quoted? Also, I'd perhaps use the `logger` for this to be in the journal.

Reply:
Personally I think calling logger in the middle of an apt hook can be dangerous.

Reply:
Thinking if you agree, to merge it as it is, with a simple echo to show info and only change in future if there is a need.